src/gui/qgsgeometryrubberband.h

```diff
@@ -106,20 +112,30 @@ class GUI_EXPORT QgsGeometryRubberBand: public QgsMapCanvasItem
     void setBrushStyle( Qt::BrushStyle brushStyle );
     //! Sets vertex marker icon type
     void setIconType( IconType iconType ) { mIconType = iconType; }
+    //! Sets whether the vertices are drawn
+    void setIsVerticesDrawn( bool isVerticesDrawn );
```

Comment:
This name sounds a bit strange - how about e.g. `setVertexDrawingEnabled( bool enabled );` ?